manifests/t/ThielickeITSolutions/TIPP10/2.1.0/ThielickeITSolutions.TIPP10.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ThielickeITSolutions.TIPP10
+PackageVersion: 2.1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.tipp10.com/en/download/tipp10_win_v2-1-0.exe
+  InstallerSha256: 405ADAF0DC8CA3C958ABD161E6DC6B682B710FFDF8AE7428153EDBED0DB6FCD3

```

```suggestion
  InstallerSha256: 405ADAF0DC8CA3C958ABD161E6DC6B682B710FFDF8AE7428153EDBED0DB6FCD3
  AppsAndFeaturesEntries:
  - DisplayName: TIPP10 Version 2.1.0
    DisplayVersion:
```
AppsAndFeaturesEntries should be added since the display version is null